frontend/lib/norent/letter-content.tsx

```diff
@@ -138,28 +168,49 @@ const LetterContentPropsFromSession: React.FC<{
   return children(lcProps);
 };
 
+const DearLandlord: React.FC<NorentLetterContentProps> = (props) => (
+  <p>
+    <Trans description="salutation of formal letter">
+      Dear <LandlordName {...props} />,
+    </Trans>
+  </p>
+);
+
+const Regards: React.FC<{ children?: React.ReactNode }> = ({ children }) => (
+  <p className="jf-signature">
+    <Trans description="before signature in formal letter">Regards,</Trans>
+    {children}
+  </p>
+);
```

Comment:
I might have gone a bit overboard with the DRY here... also, an annoying thing is that we place the remainder of stuff within the `<p>` in one situation and outside of the `<p>` in another situation because of weird differences in the way PDFs render vs. plaintext emails... blah.

Reply:
Hmmm yeah, my one worry with making each little piece of the letter content its own component is it may make it harder for the translator to visualize the whole letter together. On the other hand, I imagine that we can't just include one `<Trans>` tag around the entire letter, and the fact that you are adding these detailed descriptions should definitely help.

Reply:
Oh wait!  So uh, the funny thing is that after looking more closely at Tahnee's mockups, I realized that the to/from part of the letter doesn't actually seem to need to be translated... Here's a screenshot of her Figma mocks:

> ![image](https://user-images.githubusercontent.com/124687/83156065-f2c6a300-a0cf-11ea-87ad-8e022dd77678.png)

It looks like it only contains the part of the letter from the salutation onwards, so I guess maybe we can actually just _un_-internationalize the to/from stuff?

Reply:
Oops I replied to the wrong comment!

Reply:
Oh awesome! Ok great that solves it then :)